### DISCUSSION TITLE
新版清爽首页动态问题

### DISCUSSION THREAD

**Discussion by Alexius66** [the1812/Bilibili-Evolved#4421] [功能建议 / Ideas]
似乎动态的播放量显示存在问题，播放量过万在首页会显示为个位数
![image](https://github.com/the1812/Bilibili-Evolved/assets/73510996/880e03fa-9cbd-4b3e-8f07-c50a6b964e47)
![image](https://github.com/the1812/Bilibili-Evolved/assets/73510996/f2802795-29c0-47ef-bf1d-8a0fde536860)
首页显示的数值就是播放量万的数字

**Comment by the1812** [comment]
还真是, 测的时候没遇见过万播放的就没发现
